pkg/reconciler/vertex/controller.go

```diff
@@ -356,11 +352,18 @@ func (r *vertexReconciler) buildPodSpec(vertex *dfv1.Vertex, pl *dfv1.Pipeline,
 					},
 				},
 			})
-			podSpec.Containers[0].VolumeMounts = append(podSpec.Containers[0].VolumeMounts, corev1.VolumeMount{
-				Name:      volName,
-				MountPath: dfv1.PathPBQMount,
+		}
+		// Add emptyDir for reduce vertex pods
+		if storage.EmptyDir != nil {
```

Comment:
tiny: use `else if` to make it more readable.

Reply:
done